qiskit/pulse/schedule.py

```diff
@@ -1146,3 +1152,24 @@ def _check_nonnegative_timeslot(timeslots):
                 raise PulseError(
                     "An instruction on {} has a negative "
                     " starting time.".format(chan))
+
+
+def _get_timeslot(schedule: Union[Instruction, Schedule]) -> TimeSlots:
+    """Generate timeslot from given schedule component.
```

Comment:
I think these should be plural

```suggestion
def _get_timeslots(schedule: Union[Instruction, Schedule]) -> TimeSlots:
    """Generate timeslots from given schedule component.
```